### ISSUE TITLE
App doesnt start

### ISSUE THREAD

**Issue by rafavernet** [phase1geo/Minder#364]
I've tried to install directly with _flatpak install flathub com.github.phase1geo.minder_ and run with _flatpak run com.github.phase1geo.minder_ but after this last command the following error appears:

> ~$ flatpak run com.github.phase1geo.minder
> 
> (com.github.phase1geo.minder:2): Gtk-WARNING **: 02:33:50.071: Locale not supported by C library.
> 	Using the fallback 'C' locale.
> Gtk-Message: 02:33:50.073: Failed to load module "appmenu-gtk-module"
> Gtk-Message: 02:33:50.136: Failed to load module "canberra-gtk-module"
> Gtk-Message: 02:33:50.136: Failed to load module "canberra-gtk-module"
> 
> ** (com.github.phase1geo.minder:2): CRITICAL **: 02:33:51.952: Settings.vala:87: Could not connect: No such file or directory
> 
> ** (com.github.phase1geo.minder:2): CRITICAL **: 02:33:51.952: Settings.vala:107: Could not connect: No such file or directory
 
Also I can't rollback to 1.12.1:
~$ sudo flatpak update --commit=2824057a06a87d1f03d33cfa7b5dacbcba858b29c438463114114ff45c92d848 com.github.phase1geo.minder
Looking for updates…
error: com.github.phase1geo.minder not installed

PopOS 20.04
Flatpak version 1.8.5

**Comment by mochiballs** [comment]
Similar issue here with Linux Mint Cinnamon 19.3, after flatpak install - Minder 1.12.3 

here is the is the feedback in terminal:

```
flatpak run com.github.phase1geo.minder

** (com.github.phase1geo.minder:2): CRITICAL **: 12:11:06.455: Settings.vala:87: Could not connect: No such file or directory

** (com.github.phase1geo.minder:2): CRITICAL **: 12:11:06.456: Settings.vala:107: Could not connect: No such file or directory
```

**Comment by phase1geo** [comment]
So I attempted to build Minder from source in Linux Mint Cinnamon 20.1 and could not due to the libgranite library only having version 5.3.x.  I think I need libgranite to be version 5.5 or greater for the master branch to compile.  I'm wondering if something else is missing in Linux Mint to make this work.

**Comment by phase1geo** [comment]
FYI, Minder doesn't have a Settings.vala file in its codebase, so the error is occurring in some other code base.

**Comment by mochiballs** [comment]
> So I attempted to build Minder from source in Linux Mint Cinnamon 20.1 and could not due to the libgranite library only having version 5.3.x. I think I need libgranite to be version 5.5 or greater for the master branch to compile. I'm wondering if something else is missing in Linux Mint to make this work.

I tried searching for libgranite in the repository and found:
- libranite-common
- libgranite-dev
- libgranite4

I installed those libraries, but that didn't seem to help :/

Not sure what else to look for. Unfortunately, I'm not a coder and my knowledge about Linux dev is limited, so I'm not of much use in this regard, but I'm happy to do any legwork to help out.

**Comment by davidebocci** [comment]
Similar issue with Ubuntu Linux 20.04, Flatpak 1.11.1, Minder 1.12.3:
```
flatpak run com.github.phase1geo.minder
Gtk-Message: 10:29:22.644: Failed to load module "canberra-gtk-module"
Gtk-Message: 10:29:22.644: Failed to load module "canberra-gtk-module"

** (com.github.phase1geo.minder:2): CRITICAL **: 10:29:23.622: Settings.vala:87: Impossibile connettersi: File o directory non esistente

** (com.github.phase1geo.minder:2): CRITICAL **: 10:29:23.622: Settings.vala:107: Impossibile connettersi: File o directory non esistente
```
As you can easily understand, the final part of the message in Italian means "Could not connect: No such file or directory".

I'm not a developer, so I can't help further, but for any tests I'm available

**Comment by kgtb32** [comment]
i have the same issue when using the latest version :
flatpak version 1.6.5 Kubuntu 20.04 minder 1.12.3 
when i tries to launch Minder  i get almost the same error 
```
** (com.github.phase1geo.minder:2): CRITICAL **: 14:04:39.475: Settings.vala:87: Impossible de se connecter : Aucun fichier ou dossier de ce type

** (com.github.phase1geo.minder:2): CRITICAL **: 14:04:39.475: Settings.vala:107: Impossible de se connecter : Aucun fichier ou dossier de ce type
```
----
maybe i have a hint on that problem 
i dont know if it's linked or not, it's just a hint but i tried to clone the github project, and run the project by ./App run.
and i get something interesting 
```
../src/MapInspector.vala:282.20-282.35: error: The name `Settings' does not exist in the context of `Granite' (granite)
    var settings = Granite.Settings.get_default();
                   ^^^^^^^^^^^^^^^^
../src/MapInspector.vala:283.53-283.68: error: The name `Settings' does not exist in the context of `Granite' (granite)
    var dark     = settings.prefers_color_scheme == Granite.Settings.ColorScheme.DARK;
                                                    ^^^^^^^^^^^^^^^^
../src/MapInspector.vala:57.28-57.43: error: The name `Settings' does not exist in the context of `Granite' (granite)
    var granite_settings = Granite.Settings.get_default();
```
the settings error seen in that problem seems to come from libgranite in MapInspector

**Comment by MeztliRA** [comment]
when i tried to launch minder today the same thing happen to me, there's no Settings.vala

**Comment by MeztliRA** [comment]
i downgraded to 1.12.2 and i was getting different error and the app's window opened and then crashed. so i downgraded again, now to 1.12.1 and its working, so i guess i need to use 1.12.1 until this get fixed

**Comment by jpVm5jYYRE1VIKL** [comment]
Same on ubuntu. Feels flat package broken.

```
flatpak run com.github.phase1geo.minder

** (com.github.phase1geo.minder:2): CRITICAL **: 08:56:45.930: Settings.vala:87: Could not connect: No such file or directory

** (com.github.phase1geo.minder:2): CRITICAL **: 08:56:45.931: Settings.vala:107: Could not connect: No such file or directory
```

**Comment by ECFE** [comment]
Same experience on POP os, It seems that the flathub packaging for this is now broken. This install did work for me at some point during the last 2 months or so. Log attached below from running minder through the terminal:

`Gtk-Message: 08:53:18.783: Failed to load module "appmenu-gtk-module"
Gtk-Message: 08:53:18.874: Failed to load module "canberra-gtk-module"
Gtk-Message: 08:53:18.874: Failed to load module "canberra-gtk-module"

** (com.github.phase1geo.minder:2): CRITICAL **: 08:53:19.406: Settings.vala:87: Could not connect: No such file or directory

** (com.github.phase1geo.minder:2): CRITICAL **: 08:53:19.406: Settings.vala:107: Could not connect: No such file or directory`

**Comment by phase1geo** [comment]
I have just patched the master branch with a fix for this issue.  Could someone from this list give it a try from the master GitHub repo and let me know if this problem still exists?

**Comment by danilown** [comment]
The problem seems to be fixed.

**Comment by rafavernet** [comment]
> I have just patched the master branch with a fix for this issue. Could someone from this list give it a try from the master GitHub repo and let me know if this problem still exists?

This problem is fixed. Now works great again.
Thanks for it.

**Comment by cesar404** [comment]
Now everything works nicely (I'm using Minder on the lastest Linux Mint Cinnamon). Thanks!

**Comment by phase1geo** [comment]
I have just released 1.12.4 with this fix.
